src/features/DrugChartSlider/utils/DrugChartSliderUtils.js

```diff
@@ -102,3 +114,61 @@ export const getUTCTimeEpoch = (time, enable24HourTimers, scheduledDate) => {
   const utcTimeEpoch = moment.utc(localTime).unix();
   return utcTimeEpoch;
 };
+
+export const epochTo24HourTimeFormat = (epochSeconds) => {
+  const date = new Date(epochSeconds * 1000);
+  const hours = date.getHours().toString().padStart(2, "0");
+  const minutes = date.getMinutes().toString().padStart(2, "0");
+  const formattedTime = `${hours}:${minutes}`;
+  return formattedTime;
+};
+
+export const epochTo12HourTimeFormat = (epochSeconds) => {
```

Comment:
These methods are generic methods so move these methods to src/utils/DateTimeUtils.js file